css/advantages.scss

```diff
@@ -0,0 +1,43 @@
+@import "colors";
+
+.advantages {
+	display: flex;
+	justify-content: center;
+	text-align: center;
+
+	background-color: $secondary-color;
+
+	padding: 70px 15px 35px;
+
+	&__text-bold {
```

Comment:
Klasa sie nazywa `text-bold`, a nawet nie zmieniasz `font-weight`. Ogolnie to moim zdaniem zbyt ogolna nazwa